src/tesselation/voronoi.jl

```diff
@@ -39,9 +39,12 @@ function tesselate(pset::PointSet, method::VoronoiTesselation)
     coords = CoordRefSystems.reconstruct(C, T.(xy))
     Point(coords)
   end
-  polygs = each_polygon(vorono)
-  tuples = [Tuple(inds[begin:(end - 1)]) for inds in polygs]
-  connec = connect.(tuples)
+  polygs = get_polygons(vorono)
+  connec = Vector{Connectivity}(undef, length(polygs))
+  for (order_idx, vertices_inds) in polygs
+    tup = ntuple(i -> vertices_inds[i], length(vertices_inds) - 1)
+    connec[order_idx] = connect(tup, Ngon)
+  end
```

Comment:
Perhaps we can avoid the abstract `Connectivity` type here with an extra allocation of inds, followed by a conversion to tuple with `ntuple`. I know in this case the number of vertices in each voronoi cell is unknown, which probably leads to `Connectivity` eltype at the end anyways.

@eliascarv what do you think?